fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/serialization/LoanApplicationCommandFromApiJsonHelper.java

```diff
@@ -485,7 +485,14 @@ public void validateForCreate(final String json, final boolean isMeetingMandator
             baseDataValidator.reset().parameter(LoanApiConstants.datatables).value(datatables).notNull().jsonArrayNotEmpty();
         }
 
-        validateLoanMultiDisbursementdate(element, baseDataValidator, expectedDisbursementDate, principal);
+        if (this.fromApiJsonHelper.parameterExists(LoanApiConstants.daysInYearTypeParameterName, element)) {
+            final Integer daysInYearType = this.fromApiJsonHelper.extractIntegerNamed(LoanApiConstants.daysInYearTypeParameterName, element,
+                    Locale.getDefault());
+            baseDataValidator.reset().parameter(LoanApiConstants.daysInYearTypeParameterName).value(daysInYearType).notNull()
+                    .isOneOfTheseValues(1, 360, 364, 365);
```

Comment:
O don't know the details of this code, so this a general review which someone else with more in-depth knowledge may want to comment on, but years normally have 365 or 366 (leap year) days.. what's 364 used for, and 360 - and 1?!

Reply:
Hi @vorburger - that's the joy of archaic day count conventions for interest rate calculations :-). There are hundreds of conventions but 365 and 360 are the most common. See https://www.investopedia.com/terms/d/daycount.asp (or http://strata.opengamma.io/apidocs/com/opengamma/strata/basics/date/DayCounts.html for some more exotic ones :-)).

Reply:
@ptuomola oh what fun. OK, then I'll Resolve this.